flow/project.py

```diff
@@ -261,18 +261,14 @@ class JobOperation(object):
         :class:`dict`
     """
 
-    def __init__(self, id, name, job, cmd, directives=None):
+    def __init__(self, id, name, job, cmd, directives):
```

Comment:
Due to backwards compatibility issues, we need to allow for directives to be an optional argument still.